components/ruby/facter/Makefile

```diff
@@ -43,21 +43,16 @@ COMPONENT_LICENSE_FILE=	facter.license
 COMPONENT_POST_UNPACK_ACTION= \
     $(GEM) spec $(USERLAND_ARCHIVES)/$(COMPONENT_ARCHIVE) --ruby > $(SOURCE_DIR)/$(GEMSPEC)
 
-include $(WS_TOP)/make-rules/prep.mk
-include $(WS_TOP)/make-rules/ips.mk
+include $(WS_TOP)/make-rules/common.mk
 include $(WS_TOP)/make-rules/gem.mk
```

Comment:
I think you should use something like `BUILD_STYLE = gem` instead.

Reply:
done